include/seqan3/search/configuration/max_error_rate.hpp

```diff
@@ -37,7 +37,7 @@ namespace seqan3::search_cfg
  */
 template <typename ...errors_t>
 //!\cond
-    requires sizeof...(errors_t) <= 4 &&
+    requires (sizeof...(errors_t) <= 4) &&
             ((detail::is_type_specialisation_of_v<std::remove_reference_t<errors_t>, total> ||
               detail::is_type_specialisation_of_v<std::remove_reference_t<errors_t>, substitution> ||
               detail::is_type_specialisation_of_v<std::remove_reference_t<errors_t>, deletion>  ||
```

Comment:
Oh I can select multiple lines.
```suggestion
             ((detail::is_type_specialisation_of_v<std::remove_reference_t<errors_t>, total> ||
               detail::is_type_specialisation_of_v<std::remove_reference_t<errors_t>, substitution> ||
               detail::is_type_specialisation_of_v<std::remove_reference_t<errors_t>, deletion>  ||
```

Reply:
Here you are right, but the indentation was already wrong before my changes.